apps/src/templates/manageStudents/ConfirmRemoveStudentDialog.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {PropTypes} from 'react';
+import i18n from '@cdo/locale';
+import {Header, ConfirmCancelFooter} from '../../lib/ui/SystemDialog/SystemDialog';
+import BaseDialog from '../BaseDialog';
+import Button from "../Button";
+import color from "../../util/color";
+
+export default class ConfirmRemoveStudentDialog extends React.Component {
+  static propTypes = {
+    isOpen: BaseDialog.propTypes.isOpen,
+    disabled: PropTypes.bool,
+    onConfirm: PropTypes.func.isRequired,
+    onCancel: PropTypes.func.isRequired,
+  };
+
+  render() {
+    const {isOpen, disabled, onConfirm, onCancel} = this.props;
+    return (
+      <BaseDialog
+        useUpdatedStyles
+        isOpen={isOpen}
+        handleClose={onCancel}
+      >
+        <div style={styles.container}>
+          <Header text={i18n.removeStudentHeaderNew()}/>
+          <p>
+            <strong>{i18n.removeStudentBody1()}</strong>
```

Comment:
i just ran into this same issue (some inline text in a translated string needs to be `strong`). i think jquery's [parseHTML](https://api.jquery.com/jquery.parsehtml/) method would work, but i haven't tried it yet. any problem that you see with using that?

Reply:
nevermind... i just tried it and it seems like a very annoying solution. if it feels worth it, we can try, but otherwise it seems more complicated than necessary

Reply:
I've been holding out, hoping that the translatable markdown work @Hamms has been doing will be useful to us in places like this.  I'm hesitant to use `parseHTML` because we don't want to require our translators to be able to work with HTML, and because it looks like it might be another thing we need to take into account when we finally upgrade jQuery from 1.12 to 3.0+.

Reply:
sounds good. i'll do the same as you. i'm a little worried about how the translations will turn out when a sentence is broken into pieces, but i think that's more of a concern for the account deletion modal than this one